lib/ship_quiet_logistics/documents/shipment_order.rb

```diff
@@ -68,9 +68,9 @@ def ship_to_hash
           'Address1'   => shipment.ship_address.address1,
           'Address2'   => shipment.ship_address.address2,
           'City'       => shipment.ship_address.city,
-          'State'      => shipment.ship_address.state.name,
+          'State'      => shipment.ship_address.state_name,
```

Comment:
Also you might want to simply use [`Spree::Address#state_text`][1].

[1]: https://github.com/spree/spree/blob/2-4-stable/core/app/models/spree/address.rb#L43-L45